### PR TITLE
Fixing meta support

### PR DESCRIPTION
**Pull request by jcenturion**

### CODE REVIEW

File: lib/context.js [hunk 1]
```diff
@@ -71,6 +73,10 @@ function create(options) {
         context.data[k] = context.secrets[k] = options.secrets[k];
     }
 
+    for (let k in options.meta) {
+        context.data[k] = context.meta[k] = options.meta[k];
```
Comment (ggoodman) [inline lib/context.js]: I don't think we should put `meta` in `ctx.data`. I don't believe this is consistent w/ production.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "webtask-runtime",
-  "version": "1.6.0",
+  "version": "1.6.1",
```
Comment (ggoodman) [inline package.json]: Let's go with a minor since it's introducing a new 'feature'.

File: lib/context.js (hunk 1 above)
Comment (jcenturion) [reply lib/context.js]: Removed

File: package.json (hunk 2 above)
Comment (jcenturion) [reply package.json]: Done

**Comment by jcenturion** [comment]
@ggoodman I've already added tests :).

**Pull request merged**